subsys/bluetooth/audio/has.c

```diff
@@ -541,8 +619,61 @@ int bt_has_preset_unregister(uint8_t index)
 		return -ENOENT;
 	}
 
+	preset_changed_prepare(&buf, BT_HAS_CHANGE_ID_PRESET_DELETED, BT_HAS_IS_LAST);
+	net_buf_simple_add_u8(&buf, preset->index);
+
 	preset_free(preset);
 
+	return control_point_send_all(&buf);
+}
+
+int bt_has_preset_available(uint8_t index)
+{
+	struct has_preset *preset = NULL;
+
+	CHECKIF(index == BT_HAS_PRESET_INDEX_NONE) {
+		BT_ERR("index is invalid");
```

Comment:
Wouldn’t it be more friendly to log the invalid index value as well, to help with debugging?

Reply:
It's a valid point, although in this case there is only 1 (constant) invalid value. Logging this would be akin to logging `%p` for a NULL pointer IMO.

Reply:
Ok, fair enough, though then the log message could say something like "INDEX_NONE is not a valid value", instead of leaving the developer guessing/checking the code for what got passed. Anyway, super minor issue - I'm fine approving as-is as well.